src/hyp3_isce2/burst.py

```diff
@@ -7,8 +7,9 @@
 from pathlib import Path
 from typing import Iterator, List, Tuple, Union
 
-import pandas as pd
+import isce  # noqa
```

Comment:
Why the `# noqa` here?

Reply:
To import `isceobj`, you must first `import isce`, even if you're not directly using `isce`. Flake8 flags this as an unused import unless specify `# noqa`.